CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0
 
 All notable changes to this project will be documented in this file.
 
+## [2.3.0] - 2021-02-26
```

Comment:
Note: We shouldn't add a date here until we've actually released and merged.

Typically, what we'd do here is put the date as `UNRELEASED` and leave the version as some patch version (e.g.`2.3.0.pre.1`) while we merge to mainline.

Once we are ready to release, then we bump to `2.3.0`, add the tag, and update here with the date of the actual release.